packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/quanticBreadcrumbManager.html

```diff
@@ -21,12 +21,12 @@
             </li>
           </template>
           <template for:each={numericFacetBreadcrumbsValues} for:item="breadcrumb">
-            <li key={breadcrumb.facetId} class="slds-list_horizontal breadcrumb-manager__line">
-              <span class="slds-col breadcrumb-manager__field-name">{breadcrumb.field}{labels.colon} </span>
+            <li key={breadcrumb.facetId} class="slds-list_horizontal numericFacetBreadcrumb__list breadcrumb-manager__line">
```

Comment:
The class name should be something like `breadcrumb-manager__numeric-facet-list`.